artifact/signer.go

```diff
@@ -19,10 +19,10 @@ import (
 	"crypto/ecdsa"
 	"crypto/rand"
 	"crypto/rsa"
-	"github.com/minio/sha256-simd"
 	"crypto/x509"
 	"encoding/base64"
 	"encoding/pem"
+	"github.com/minio/sha256-simd"
```

Comment:
Minor comment: can you please move this line to the second group of imports to group github.com items together?